docker-support/Dockerfile

```diff
@@ -8,6 +8,7 @@ RUN apk update && \
 	make \
 	nodejs \
 	npm \
+	coreutils \
```

Comment:
#266 has been merged, so I assume we won't need this - but it won't hurt either I assume...

Reply:
Next time someone tries to use something advanced and unsupported by busybox for any command, we won't trip over it again. 👍 I'm not sure anyone would have noticed but I just made this and so, search and destroy error messages.

There is no urgency to merge this or to update the website support image.